TYPO3.Flow/Classes/TYPO3/Flow/Property/TypeConverter/DateTimeConverter.php

```diff
@@ -143,7 +143,8 @@ public function convertFrom($source, $targetType, array $convertedChildPropertie
         if ($dateAsString === '') {
             return null;
         }
-        if (ctype_digit($dateAsString) && $configuration === null && (!is_array($source) || !isset($source['dateFormat']))) {
+        $configurationForm = $configuration === null ? null : $configuration->getConfigurationValue('TYPO3\Flow\Property\TypeConverter\DateTimeConverter', self::CONFIGURATION_DATE_FORMAT);
```

Comment:
Thx for the quick feedback, that was a typo actually and should have read `configurationForm*at*`.
I was under the (false?) impression that 3.0 should still work with PHP < 5.5, so ::class was avoided, but I'm probably mixing things up based on the fact that 3.0 still contains lots of FQCN strings instead of ::class.

Otherwise I'm with you on making that part more explicit, I'll push a follow-up.